stratum/hal/lib/tdi/dpdk/dpdk_sde_target.cc

```diff
@@ -244,29 +250,6 @@ ::util::Status TdiSdeWrapper::DisablePort(int device, int port) {
   return MAKE_ERROR(ERR_UNIMPLEMENTED) << "DisablePort not implemented";
```

Comment:
Note that DPDK does not support EnablePort or DisablePort

DPDK also does not support the four-argument AddPort method. (This method is Tofino-specific.)
It supports the three-argument AddPort method (which is DPDK-specific).

Reply:
Note that I introduced a generic 2 parameter AddPort within this change in the TdiPortManager interface.
The expectation is every target implements these base methods mandatorily.
If you do not intend to make use of the Enable/Disable APIs, you could either return an ERR_UNIMPLEMENTED or OkStatus().

Another option is to introduce an intermediate DummyPortManager implementation and have that implement the mandatory methods and then derive off that class. I think that is unnecessary since the number of required methods is fewer than 10.